### DISCUSSION TITLE
重采样伪影怎么没找到

### DISCUSSION THREAD

**Discussion by hkiceflow** [hooke007/mpv_PlayKit#352] [回收]
我在文章中看见了重采样伪影
相关列表：
Anime4K_RA_CNN_M.glsl
Anime4K_RA_CNN_L.glsl
Anime4K_RA_CNN_UL.glsl
Anime4K_RA_DoG.glsl

但是我在shaders中并没有看见相关变体。

**Comment by hooke007** [comment]
a4kv3.1的介绍资料是只是归档备份，现在早就v4了

**Reply by hkiceflow** [reply]
懂了 谢谢
